FlinkSinkWriterTest.testSinkMetrics is unstable

### Search before asking

- [x] I searched in the [issues](https://github.com/alibaba/fluss/issues) and found nothing similar.


### Fluss version

main (development)

### Please describe the bug 🐞

https://github.com/alibaba/fluss/actions/runs/13721569801/job/38378010397?pr=548

```
Error:  Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 14.698 s <<< FAILURE! - in com.alibaba.fluss.connector.flink.sink.FlinkSinkWriterTest
Error:  com.alibaba.fluss.connector.flink.sink.FlinkSinkWriterTest.testSinkMetrics(String)[1]  Time elapsed: 0.08 s  <<< FAILURE!
java.lang.AssertionError: 

Expecting actual:
  0L
to be greater than:
  0L

	at com.alibaba.fluss.connector.flink.sink.FlinkSinkWriterTest.testSinkMetrics(FlinkSinkWriterTest.java:109)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:4[98](https://github.com/alibaba/fluss/actions/runs/13721569801/job/38378010397?pr=548#step:4:99))
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:[95](https://github.com/alibaba/fluss/actions/runs/13721569801/job/38378010397?pr=548#step:4:96))
	at org.junit.platform.engine.support.hierarchical.ForkJoinPoolHierarchicalTestExecutorService$ExclusiveTask.compute(ForkJoinPoolHierarchicalTestExecutorService.java:185)
	at java.util.concurrent.RecursiveAction.exec(RecursiveAction.java:189)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)

[INFO] Running com.alibaba.fluss.connector.flink.row.FlinkAsFlussRowTest
[INFO] Tests run: 16, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.013 s - in com.alibaba.fluss.connector.flink.row.FlinkAsFlussRowTest
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 17.53 s - in com.alibaba.fluss.connector.flink.source.lookup.FlinkLookupFunctionTest
[INFO] 
[INFO] Results:
[INFO] 
Error:  Failures: 
Error:    FlinkSinkWriterTest.testSinkMetrics:109 
Expecting actual:
  0L
to be greater than:
  0L

[INFO] 
Error:  Tests run: 60, Failures: 1, Errors: 0, Skipped: 0
```

### Solution

_No response_

### Are you willing to submit a PR?

- [ ] I'm willing to submit a PR!

--- a/fluss-client/src/main/java/com/alibaba/fluss/client/metrics/WriterMetricGroup.java
+++ b/fluss-client/src/main/java/com/alibaba/fluss/client/metrics/WriterMetricGroup.java
@@ -42,9 +42,8 @@ public class WriterMetricGroup extends AbstractMetricGroup {
     private final Histogram bytesPerBatch;
     private final Histogram recordPerBatch;
 
-    private volatile long sendLatencyInMs;
-
-    private volatile long batchQueueTimeMs;
+    private volatile long sendLatencyInMs = -1;
+    private volatile long batchQueueTimeMs = -1;
 
     public WriterMetricGroup(ClientMetricGroup parent) {
         super(parent.getMetricRegistry(), makeScope(parent, name), parent);

--- a/fluss-connectors/fluss-connector-flink/src/test/java/com/alibaba/fluss/connector/flink/metrics/FlinkMetricsITCase.java
+++ b/fluss-connectors/fluss-connector-flink/src/test/java/com/alibaba/fluss/connector/flink/metrics/FlinkMetricsITCase.java
@@ -127,8 +127,8 @@ class FlinkMetricsITCase extends FlinkTestBase {
         assertThat(metricsList).hasSize(1);
         Metric sendLatencyMetrics = metricsList.get(0).f2;
         assertThat(sendLatencyMetrics).isInstanceOf(Gauge.class);
-        // just check send latency is greater than 0
-        assertThat((Long) ((Gauge<?>) sendLatencyMetrics).getValue()).isGreaterThan(0);
+        // the default send latency is -1, so check it is >= 0, as the latency maybe very small 0ms
+        assertThat((Long) ((Gauge<?>) sendLatencyMetrics).getValue()).isGreaterThanOrEqualTo(0);
 
         // test scan
         tableResult = tEnv.executeSql("select * from test");

--- a/fluss-connectors/fluss-connector-flink/src/test/java/com/alibaba/fluss/connector/flink/sink/FlinkSinkWriterTest.java
+++ b/fluss-connectors/fluss-connector-flink/src/test/java/com/alibaba/fluss/connector/flink/sink/FlinkSinkWriterTest.java
@@ -106,7 +106,8 @@ public class FlinkSinkWriterTest extends FlinkTestBase {
 
         Metric currentSendTime = interceptingOperatorMetricGroup.get(MetricNames.CURRENT_SEND_TIME);
         assertThat(currentSendTime).isInstanceOf(Gauge.class);
-        assertThat(((Gauge<Long>) currentSendTime).getValue()).isGreaterThan(0);
+        // the default send latency is -1, so check it is >= 0, as the latency maybe very small 0ms
+        assertThat(((Gauge<Long>) currentSendTime).getValue()).isGreaterThanOrEqualTo(0);
 
         Metric numRecordSend = interceptingOperatorMetricGroup.get(MetricNames.NUM_RECORDS_SEND);
         assertThat(numRecordSend).isInstanceOf(Counter.class);

